### PR TITLE
render correctly and rspec tests for 403 page

### PR DESCRIPTION
**Pull request by autotelik**
This PR adds in correct handling for 403 

Correct partial inclusion for error view
Add associated snippets to locale en.yml
Add request and feature tests to rspec to exercise 403

### CODE REVIEW

File: spec/features/error_pages_spec.rb [hunk 1]
```diff
@@ -15,6 +15,16 @@ module DigitalServicesCore
       expect(page).to have_text( I18n.t('global.notfound.message') )
     end
 
+    context '403' do
+
+      #TOFIX - really we want to visit a real controller action that in unauthorised
```
Comment (Cruikshanks) [inline spec/features/error_pages_spec.rb]: Minor thing but can this be changed to start # TODO: FIX - TODO is recognised by a number of tools and should therefore help us keep track of them going forward. What do you think to also raising an issue (with the label *enhancement*) in github to highlight this is something we'd like to resolve and ensure we don't forget about it?

File: spec/requests/error_pages_spec.rb [hunk 2]
```diff
@@ -1,8 +1,52 @@
 require 'spec_helper'
 
-
 describe 'custom error handling for bad requests',  :type => :request do
 
+  context '403' do
+
+=begin
+#TODO - How to test a real 403 situation ???
```
Comment (Cruikshanks) [inline spec/requests/error_pages_spec.rb]: Same as previous. Can this be changed to start # TODO: as it is recognised by a number of tools and should therefore help us keep track of them going forward. Also the idea of raising an issue to keep track of things like this we'd like to fix, but I would combine a solution to this and the one above in the same issue.

**Comment by Cruikshanks** [comment]
Might be a dumb question but as a rails engine how would I go about executing the tests locally. Is it just a case of pulling it down and running ```rspec``` as normal? I'd like to do that locally before signing it off.

**Comment by autotelik** [comment]
Sure, happy to use that TODO form.. I just used those original forms as Rubymine recognises them but it also recognises "# TODO:" .. with the colon right ?

And yes, personally I really like working in github so more than happy to use it for issue tracking

**Comment by autotelik** [comment]
no dumb questions when it comes to rails ! ;-)  .. so within the spec dir is a dummy/skeleton rails app that is used as a container app for the engine for testing. you can also run up the server in there and view stuff in actual browser. So yes, you should be able to simply pull it down locally and run rspec  .. famous last words !

**Comment by autotelik** [comment]
As Alan away, gonna close this PR myself, so we can continue with new features off develop

**Pull request merged**